tgui/packages/tgfont/README_ICON_TUTORIAL.txt

```diff
@@ -0,0 +1,26 @@
+The following is the process to implement your own icon using an svg.
+
+If you plan on making your own SVG, i suggest Inkscape, as it is free and pretty powerful for vector graphics.
+
+1. Get whatever SVG you plan on using and put it in the ..\tgstation\tgui\packages\tgfont\icons folder.
+
+2. In VScode, at the top, open "Terminal>New Terminal" and run this exact line of code without the quotes -> ".\BUILD.cmd tg-font"
+    Wait till its all complete
+
+3. In ..\tgstation\tgui\packages\tgfont\dist you will find 3 files that were generated, a .css, .eot and a .WOFF2 file. Copy and replace all 3 of these files to
+    the following directory -> ..\tgstation\tgui\packages\tgfont\static
+
+4. Then we run/double click the BUILD.bat file in ..\tgstation\
```

Comment:
wait why are you doing any of this. you should be able to just build. if you cant that is a bug

Reply:
> wait why are you doing any of this. you should be able to just build. if you cant that is a bug

Ask Anturk, hes the one managing the system and I followed his steps.

Reply:
@AnturK theres no way you need to copy and replace files for tgfont bro